charts/loki-distributed/values.yaml

```diff
@@ -181,6 +181,9 @@ serviceMonitor:
   interval: null
   # -- ServiceMonitor scrape timeout in Go duration format (e.g. 15s)
   scrapeTimeout: null
+  # -- ServiceMonitor relabel configs to apply to samples before scraping
+  # https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#relabelconfig
+  relabelings: null
```

Comment:
A list is expected, so please make this and empty list `[]`.

Reply:
Fixed